man/org.freedesktop.systemd1.xml

```diff
@@ -1685,6 +1685,9 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly b IgnoreOnIsolate = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly b NeedDaemonReload = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      @org.freedesktop.systemd1.Privileged("true")
+      readwrite b NeedsRestart = ...;
```

Comment:
I thought about this, but the the old naming is broken :( "NeedRestart" is just wrong. This is supposed to be an attribute so "this unit needs a restart", "this unit needs a daemon reload". Without the "s" it doesn't make sense gramatically.

Reply:
Actually, "NeedDaemonReload" is okay-ish. Because even though it's a unit property, it applies to the whole manager, because we can't yet reload individual units. So "we need a daemon reload" can be abbreviated to "NeedDaemonReload", maybe. But the same doesn't hold for NeedsRestart.